test/tools/isoltest.cpp

```diff
@@ -401,20 +402,26 @@ int main(int argc, char const *argv[])
 {
 	setupTerminal();
 
-	solidity::test::IsolTestOptions options(&TestTool::editor);
-
-	try
 	{
-		if (options.parse(argc, argv))
-			options.validate();
-		else
+		using IOptions = solidity::test::IsolTestOptions;
```

Comment:
It might be more useful to add `using namespace solidity::test` at the top of this file.

Reply:
this one is actually only used once, so I just use it directly here